integration_tests/integration/page-not-found.spec.ts

```diff
@@ -0,0 +1,38 @@
+import FindRecipientByPrisonNumberPage from '../pages/barcode/findRecipientByPrisonNumber'
+import Error404Page from '../pages/error404'
+import Page from '../pages/page'
+import RequestOneTimeCodePage from '../pages/one-time-code/requestOneTimeCode'
+
+context('404 Page Not Found', () => {
+  beforeEach(() => {
+    cy.task('reset')
+  })
+
+  describe('Legal Sender user', () => {
+    it(`should show the 'page not found' page when authenticated as a legal sender and navigating to a non existent SLM url`, () => {
+      FindRecipientByPrisonNumberPage.goToPage()
+
+      cy.visit('/barcode/find-recipient-by-inside-leg-measurement', { failOnStatusCode: false })
+
+      Page.verifyOnPage(Error404Page)
+    })
+
+    it(`should redirect to the OTC sign in page when not authenticated and navigating to a non existent SLM url`, () => {
+      cy.visit('/barcode/find-recipient-by-inside-leg-measurement', { failOnStatusCode: false })
+
+      Page.verifyOnPage(RequestOneTimeCodePage)
+    })
+  })
```

Comment:
See the change in `nunjucksSetup.ts` 👇 , but there is another use case that we cannot test here
That is where we have an unauthenticated user, and they hit a non-existent url on the SLM domain.
eg: `https://send-legal-mail-to-prisons..../some-non-existent-page`
In that case we would also expect the 404 page, with the SLM page heading
But we cant test that because cypress runs the app up as localhost